srai/utils/constants.py

```diff
@@ -0,0 +1,3 @@
+"""Constants used across the project."""
+
+CRS = "EPSG:4326"
```

Comment:
Maybe we can change it to `WGS84_CRS` to make it more verbose.

Reply:
I thought about having a default CRS for our library, so I didn't want to include its name in the constant. Do you think that such a default is a good idea or is it some overengineering?

Reply:
I think it's not a very big change, but for everyone looking at the code from the outside, it'll be obvious which CRS we are using, without outside knowledge written in documentation or the readme file.

Reply:
Fair point, you convinced me